dev_docs/07_visual_testing_guide.md

```diff
@@ -0,0 +1,128 @@
+# Visual Testing
+
+KDS has a visual testing system that allows you to take snapshots of how KDS Components would look like in different browsers, and compare them with the previously set baseline versions of these components, allowing you to see the visual differences in a PR's changes more quickly.
+
+## Prerequisites
+
+- Make sure that your Node version is > 18.x with all the requirements (listed in package.json) installed.
+- Ensure that you declare the `PERCY_TOKEN` environment variable when running visual tests locally.
+- You should not have anything running on `port:4000` as the visual testing server utilizes that port to run the tests.
+
+## Guide to using the visual testing mechanism
+
+### Running visual tests locally
+
+1. **Setup**:
+   - Ensure all dependencies are installed:
+
+     ```bash
+     yarn install
+     ```
+
+   - Set the `PERCY_TOKEN` environment variable:
+
+     ```bash
+     export PERCY_TOKEN=<your-percy-token>
+     ```
+
+2. **Run Visual Tests**:
+   - Execute the tests with the visual testing configuration:
+
+     ```bash
+     yarn test:visual
+     ```
+
+3. **Check results**:
+   - Head over to the Percy Dashboard and check the results for the latest Percy build.
+
+### Visual testing workflow
+
+  We use GitHub Actions to execute the visual testing workflow on every PR. When changes to a PR are made, a deployment request is initiated. Only **Learning Equality team members** can approve this request. Once approved, the deployment is executed, and visual tests are run automatically. The results of the test run are surfaced in the form of an automated comment containing a link to the Percy build.
+
+  **Note:** Developers outside of Learning Equality can only review the visual changes for local test runs. To review the visual tests executions that run on GitHub Actions, one needs to be a part of Learning Equality's Browserstack team.
+
+### Writing visual tests
+
+  You can write the visual tests alongside the unit tests, i.e. in the same test file. Take a look at [`KButton.spec.js`](../lib/buttons-and-links/__tests__/KButton.spec.js) to familiarize yourself with writing visual tests.
+
+1. **Import Utility Functions**:
+   - Import the utility functions from [`visual.testUtils.js`](../jest.conf/visual.testUtils.js):
+
+     ```javascript
+     import { renderComponent, takeSnapshot } from './visual.testUtils';
+     ```
+
+     You can also import and use the utility functions for managing component's visual states as needed. Different utility functions available are:
```

Comment:
I would remove the "also" as below there are the `renderComponent` and `takeSnapshot` methods that are also mentioned in the code example.
```suggestion
You can import and use the utility functions for managing component's visual states as needed. Different utility functions available are:
```